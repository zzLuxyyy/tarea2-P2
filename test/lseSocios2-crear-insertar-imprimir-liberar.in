crearLSESocios
imprimirLSESocios
liberarLSESocios
crearLSESocios
crearSocio 12345678 Mickey Mouse 18 11 1928
agregarGeneroFavoritoSocio 1
insertarLSESocios
imprimirLSESocios
liberarLSESocios
Fin
