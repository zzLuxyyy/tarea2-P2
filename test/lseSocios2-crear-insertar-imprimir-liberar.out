1>Se ejecutó crearTLSESocios exitosamente.
2>Lista de Socios:
3>4>Se ejecutó crearTLSESocios exitosamente.
5>Socio de ci 12345678 creado exitosamente.
6>Se ejecutó 'agregarGeneroFavoritoTSocio' con idGenero 1 exitosamente.
7>Socio de CI 12345678 agregado a la lista de socios de forma exitosa.
8>Lista de Socios:
Socio 12345678: Mickey Mouse
Fecha de alta: 18/11/1928
Géneros favoritos: 1 
9>10>Fin.
